example/build.gradle

```diff
@@ -11,7 +11,7 @@ android {
   properties.load(file('../local.properties').newDataInputStream())
 
   defaultConfig {
-    applicationId "com.moengage.segment.example"
+    applicationId "com.moengage.sampleapp"
```

Comment:
This change is not required. Leave the example as is.

Reply:
fixed